docs/book/src/topics/managedcluster.md

```diff
@@ -102,6 +102,15 @@ spec:
   version: v1.21.2
   networkPolicy: azure # or calico
   networkPlugin: azure # or kubenet
+  sku: Free # or Paid
+  loadBalancerProfile: # Load balancer profile must specify at most one of ManagedOutboundIPs, OutboundIPPrefixes and OutboundIPs
+    managedOutboundIPs: 2 # 1-100
+    outboundIPPrefixes:
+    - /subscriptions/fae7cc14-bfba-4471-9435-f945b42a16dd/resourceGroups/foo-bar/providers/Microsoft.Network/publicIPPrefixes/my-public-ip-prefix # fake public ip prefix
```

Comment:
@richardchen331 can you please remove the sub ID from the example? replace it by something that is obviously fake like `123`

Reply:
sure, replaced it with `00000000-0000-0000-0000-000000000000` :)